Update multipanel.py to save orientation

vganapati opened the pull request:
Update multipanel.py to save the orientation in the H5 file.

vganapati committed:
Update multipanel.py

Update multipanel.py to save the orientation in the H5 file

Baharis reviewed (approved):
Other than preferences (I'd probably specify explicitly when `convert_numpy=True`, not `False`), I see no issues.

nksauter reviewed (changes requested):
I'd like to ask for a few changes,
1) All instances of the word "rotation" in the file should be changed to "Umatrix".
2) The save_variable() function should be moved outside of the run_sim2h5 method.
3) It's alarming that the save_variable() function deletes data already written.  Is it not possible to create the dataset if it does not exist, but use "append" if it exists?

nksauter committed:
Some reorganization for PR#28.

Change instances of "rotation" to "Umatrix"
Move save_variable function outside the run_sim2h5 method
Express certain caveats as comments.

nksauter committed:
Bugfix.  Must use "rotation" for compatibility.

Keep the positional argument for "rotation".  When saving it to H5, call
it "Umatrix_rot".

nksauter committed:
Bugfix.

nksauter committed:
Bugfix.

vganapati commented:
The deletion instead of appending is a hack, if a better way can be found in the future, this should be changed.
